src/mito2/src/request.rs

```diff
@@ -550,6 +550,11 @@ impl WorkerRequest {
                 sender: sender.into(),
                 request: DdlRequest::Truncate(v),
             }),
+            RegionRequest::Catchup(v) => WorkerRequest::Ddl(SenderDdlRequest {
```

Comment:
I think the term "DDL" is overused. Catchup (also Compact, Flush, Truncate) is not very much into the degree of "Data Definition Language". How about create a new category for these kinds of request, called "DclRequest"? DCL = Data Control Language
WDYT @evenyag

Reply:
Yes, I agree this. Let’s refactor it in the another PR.